### PR TITLE
ADD updateEmail

### PR DESCRIPTION
**Pull request by elco45**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ Make sure your client provides a implementation of [`fetch`](https://developer.m
 - [`*auth.signOut()`](https://n6g7.github.io/redux-saga-firebase/reference/auth#signOut)
 - [`*auth.unlink(authProvider)`](https://n6g7.github.io/redux-saga-firebase/reference/auth#unlink)
 - [`*auth.updatePassword(password)`](https://n6g7.github.io/redux-saga-firebase/reference/auth#updatePassword)
+- [`*auth.updateEmail(email)`](https://n6g7.github.io/redux-saga-firebase/reference/auth#updateEmail)
```
Comment (n6g7) [inline README.md]: Can you keep the alphabetical order please? :)

File: docs/_reference/auth.md [hunk 2]
```diff
@@ -458,4 +458,26 @@ methods:
...
+      function* updateEmailSaga(email) {
+        try {
+          yield call(rsf.auth.updateEmail, email);
+          yield put(updateEmail());
+        }
+        catch(error) {
+          yield put(updateEmailFailure(error));
+        }
+      }
+      ```
```
Comment (n6g7) [inline docs/_reference/auth.md]: same

**Comment by n6g7** [comment]
@elco45 I just release [v0.13.0](https://github.com/n6g7/redux-saga-firebase/releases/tag/v0.13.0) with your changes, thanks again!

**Comment by elco45** [comment]
@n6g7 Thank you for creating this! It is really awesome and useful!

**Pull request merged**